docs/app/pages/components/components-sections/badge/badge/badge.component.html

```diff
@@ -0,0 +1,69 @@
+<p>Demo here</p>
+
+<hr>
+
+<p>
+    The <code>ux-badge</code> directive can be used to easily attached a badge to any DOM element to be used as a label/tag, or to show (x) number of new items relating to it etc.
+</p>
+
+<p>The appearance and behavior of the directive can be configured using the following attributes:</p>
+
+<uxd-api-properties tableTitle="Inputs">
+    <tr uxd-api-property name="badgeBackgroundColor" type="string">
+        The color you want to use for the background color. This should be a string value that matches one of the colors available in the <code>ColorService</code>.
+    </tr>
+    <tr uxd-api-property name="badgeTextColor" type="string">
+        The color you want to use for the text/number on the badge. This should be a string value that matches one of the colors available in the <code>ColorService</code>. If you do not set this value the <code>ContrastService</code> will automatically pick which text color is suitable based on the <code>badgeBackgroundColor</code>.
+    </tr>
+    <tr uxd-api-property name="badgeText" type="string">
+        The text you wish to display on the badge. This property being set will trigger a check to detect if it is a number or text.
+    </tr>
+    <tr uxd-api-property name="badgeMaxLength" type="number">
+        This allows you to set a character / number limit on the text within the badge. If the content is number and larger than the specified size it will append a <strong>+</strong> otherwise it will the content as text and truncate the characters using ellipsis <strong>&hellip;</strong>.
```

Comment:
I'll change it to `badgeValueLimit` and the behaviour will work like so: if a number is passed it will show this number if the value exceeds it followed by a '+' (as you mentioned above), if text this number is used as a char length for the string and truncates.

Reply:
Yes, that sound fine, thank you.